sql.py

```diff
@@ -75,6 +75,23 @@ async def get_all_patreons(pool):
             data = await cursor.fetchall()
             return data
 
+async def change_username(pool, uid, newname):
+    async with pool.acquire() as conn:
+        async with conn.cursor() as cursor:
+            sql = "SELECT * from rotmg.users WHERE id = %s AND status = 'verified'"
+            data = (uid)
+            await cursor.execute(sql, data)
+            data = await cursor.fetchone()
+            if not data:
+                return False
+            if data[usr_cols.ign]:
+                return False
+            sql = f"UPDATE rotmg.users SET ign = %s WHERE id = %s"
+            data = (newname, uid)
+            await cursor.execute(sql, data)
+            await conn.commit()
+            return True
```

Comment:
Instead of making 2 calls to the server, i'd go with a single update or insert statement (something like):
`sql = "IF IF EXISTS(select * from rotmg.users where id=%s)
   update test set ign=%s where uid=%s
ELSE
   insert into rotmg.users (id, ign) values (%s, %s);`